src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/FoundIndication.java

```diff
@@ -59,7 +58,6 @@
     private String pattern;
     private Run build;
     private String matchingString;
-    private FailureCause cause;
```

Comment:
@rsandell @oleg-nenashev removing this stuff produces
```
SEVERE: Failed Loading plugin build-failure-analyzer
java.io.IOException: Failed to initialize
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:442)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:406)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$8.runTask(Jenkins.java:926)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Unable to read /var/lib/jenkins/build-failure-analyzer.xml
	at hudson.XmlFile.unmarshal(XmlFile.java:165)
	at hudson.Plugin.load(Plugin.java:264)
	at com.sonyericsson.jenkins.plugins.bfa.PluginImpl.start(PluginImpl.java:151)
	at hudson.ClassicPluginStrategy.startPlugin(ClassicPluginStrategy.java:450)
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:439)
	... 9 more
Caused by: com.thoughtworks.xstream.converters.ConversionException: null : null
---- Debugging information ----
cause-exception     : java.lang.NullPointerException
cause-message       : null
class               : com.sonyericsson.jenkins.plugins.bfa.PluginImpl
required-type       : com.sonyericsson.jenkins.plugins.bfa.PluginImpl
converter-type      : hudson.util.RobustReflectionConverter
path                : /com.sonyericsson.jenkins.plugins.bfa.PluginImpl
--
java.io.IOException: Unable to load org.jenkinsci.plugins.globalEventsPlugin.GlobalEventsPlugin from groovy-events-listener-plugin
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:425)
	at hudson.PluginManager$2$1$1.run(PluginManager.java:406)
	at org.jvnet.hudson.reactor.TaskGraphBuilder$TaskImpl.run(TaskGraphBuilder.java:169)
	at org.jvnet.hudson.reactor.Reactor.runTask(Reactor.java:282)
	at jenkins.model.Jenkins$8.runTask(Jenkins.java:926)
	at org.jvnet.hudson.reactor.Reactor$2.run(Reactor.java:210)
	at org.jvnet.hudson.reactor.Reactor$Node.run(Reactor.java:117)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ExceptionInInitializerError
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at hudson.ClassicPluginStrategy.load(ClassicPluginStrategy.java:419)
	... 9 more
```
I restored this line (and from `Indication`) and everything works again. So, what should I do? Restore it? And mark `transient`?